K2Bridge/Visitors/Aggregations/HistogramAggregationVisitor.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.Visitors
+{
+    using System;
+    using K2Bridge.Models.Request.Aggregations;
+    using K2Bridge.Models.Response;
+    using K2Bridge.Utils;
+
+    /// <content>
+    /// A visitor for the <see cref="HistogramAggregation"/> element.
+    /// Sample KustoQL Query:
+    /// | where ['AvgTicketPrice'] between (50 .. 150);
+    /// (_data | summarize count() by ['key%False'] = bin(['AvgTicketPrice'], 20)
+    /// | where ['count_'] >= 0
+    /// | order by ['key%False'] asc | as aggs);
```

Comment:
This query isn't valid... All the changes below are meant to get to this query:

```
(_data
| where ['AvgTicketPrice'] between (50 .. 150)
| summarize count() by ['key%False'] = bin(['AvgTicketPrice'], 20)
| where ['count_'] >= 0
| order by ['key%False'] asc ) as aggs;
```

Is this the right query?

Reply:
Yes with a very small fix:
```kusto
(_data
| where ['AvgTicketPrice'] between (50 .. 150)
| summarize count() by ['key%False'] = bin(['AvgTicketPrice'], 20)
| where ['count_'] >= 0
| order by ['key%False'] asc | as aggs);
```

Reply:
@tamirkamara this query is not guiving correct results based on the e2e tests, the hard bounds need to delimit the calculated values stored in ['key%False'], which is not the case here. The correct query is the very first one:
```kusto
(_data 
| summarize count() by ['2%False'] = bin(['AvgTicketPrice'], 20)
| where ['2%False'] between (50 .. 150)
| where ['count_'] >= 1
| order by ['2%False'] asc | as aggs);
```